### PR TITLE
release 1.5

### PR DESCRIPTION
**Pull request by andyindahouse**
- [ ] TODO: After merge this, create a git-tag with 1.5.0 version

### CODE REVIEW

File: rider-app/ios/App/App/Info.plist [hunk 1]
```diff
@@ -5,7 +5,7 @@
 	<key>CFBundleDevelopmentRegion</key>
 	<string>en</string>
 	<key>CFBundleDisplayName</key>
-	<string>Kiwi Shopper</string>
+	<string>Kiwi Shopper Dev</string>
```
Comment (andyindahouse) [inline rider-app/ios/App/App/Info.plist:8]: undo this change

File: rider-app/ios/App/App.xcodeproj/project.pbxproj [hunk 2]
```diff
@@ -351,7 +351,7 @@
 				IPHONEOS_DEPLOYMENT_TARGET = 11.0;
 				LD_RUNPATH_SEARCH_PATHS = "$(inherited) @executable_path/Frameworks";
 				OTHER_SWIFT_FLAGS = "$(inherited) \"-D\" \"COCOAPODS\" \"-DDEBUG\"";
-				PRODUCT_BUNDLE_IDENTIFIER = io.kiwishopper.starter;
+				PRODUCT_BUNDLE_IDENTIFIER = io.kiwishopper.develop;
```
Comment (andyindahouse) [inline rider-app/ios/App/App.xcodeproj/project.pbxproj:354]: undo this change

**Review by SergioMorchon — APPROVED** [review]
